charts/tempo-distributed/templates/compactor/deployment-compactor.yaml

```diff
@@ -54,6 +54,8 @@ spec:
       hostAliases:
         {{- toYaml . | nindent 8 }}
       {{- end }}
+      initContainers:
+        {{- toYaml .Values.ingester.initContainers | nindent 8 }}
```

Comment:
This is a bug.

It should have been `{{- toYaml .Values.compactor.initContainers | nindent 8 }}`

Reply:
Will be fixed in this PR: https://github.com/grafana/helm-charts/pull/3552 , thx!